tests/conftest.py

```diff
@@ -23,51 +23,67 @@ def path_test_file(cwd):
 
 
 @pytest.fixture(scope='session')
-def serpent_depcode(cwd):
-    """SerpentDepcode object for unit tests"""
+def serpent_runtime(cwd, tmpdir_factory):
+    """SaltProc objects for Serpent unit tests"""
     saltproc_input = str(cwd / 'serpent_data' / 'tap_input.json')
     _, _, _, object_input = read_main_input(saltproc_input)
```

Comment:
We've discussed this before, I think. I don't like the practice of using underscores as variable names. They should have names, even if you're not using them. Also, consider that **you** might forget the return variables are for this function. Yes you could look it up, but why waste the energy?

Reply:
Okay, I think I can address this in a way that satisfies both of us.